src/dynapi/SDL_dynapi_procs.h

```diff
@@ -855,6 +855,7 @@ SDL_DYNAPI_PROC(bool,SDL_SetHapticGain,(SDL_Haptic *a, int b),(a,b),return)
 SDL_DYNAPI_PROC(bool,SDL_SetHint,(const char *a, const char *b),(a,b),return)
 SDL_DYNAPI_PROC(bool,SDL_SetHintWithPriority,(const char *a, const char *b, SDL_HintPriority c),(a,b,c),return)
 SDL_DYNAPI_PROC(void,SDL_SetInitialized,(SDL_InitState *a, bool b),(a,b),)
+SDL_DYNAPI_PROC(void,SDL_SetRelativeMouseTransform,(const void *a, void *b),(a,b),)
```

Comment:
This should not be sorted now that the ABI is locked. Best drop dynapi changes from this PR. We'll regenerate it with the other new functions in 3.4.